### PR TITLE
feat: add "registry" option for the deploy executor

### PR DESCRIPTION
**Pull request by dmitry-stepanenko**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [x] Both workspaces were tested Angular and Nx (for bug fixes/features)

## What is the current behavior?
Right now there's no ability to deploy to other registries. This might be useful if we want to add another deployment configuration for testing purposes

Issue Number: N/A

## What is the new behavior?
Adding the new `--registry` options for the deploy executor. This will allow to deploy the package into other registries or tools like https://verdaccio.org/docs/what-is-verdaccio/
## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -302,7 +310,7 @@ configuration in the `workspace.json` file in the `options` attribute
 of your deploy project's executor.
 Just change the option to lower camel case.
 
-A list of all available options is also available [here](https://github.com/bikecoders/ngx-deploy-npm/blob/main/src/deploy/schema.json).
+A list of all available options is also available [here](https://github.com/bikecoders/ngx-deploy-npm/blob/main/packages/ngx-deploy-npm/src/executors/deploy/schema.json).
```
Comment (dianjuar) [inline README.md:313]: Thank you!

**Review by dianjuar — APPROVED** [review]
Awesome contribution.
Thank you!

**Comment by dianjuar** [comment]
Interesting feature!
 
I could use this feature and verdaccio in my integration test to make them more realistic.

**Comment by dianjuar** [comment]
Feature available on [ngx-deploy-npm/v/5.2.0](https://www.npmjs.com/package/ngx-deploy-npm/v/5.2.0)

**Comment by dmitry-stepanenko** [comment]
@dianjuar thanks for fixing the unit test and merging it!

**Comment by dmitry-stepanenko** [comment]
> Interesting feature!
> 
> I could use this feature and verdaccio in my integration test to make them more realistic.

Totally makes sense, I was considering doing the same with the package I'm working on 🙁

**Pull request merged**